### PR TITLE
Add active deadline to browse pod page

### PR DESCRIPTION
**Pull request by spadgett**
openshift_web_console

Suggested by @derekwaynecarr 

@jwforres PTAL

### CODE REVIEW

**Comment by derekwaynecarr** [comment]
Nice!  Even cooler if you add a edit button and let it shrink ;-)

**Comment by spadgett** [comment]
@derekwaynecarr do you mean a countdown?

**Comment by derekwaynecarr** [comment]
In kube 1.2, you can make the value go from unset to set, and if set to less than its previous set value.

**Comment by derekwaynecarr** [comment]
not a huge priority for me, i am just happy to see the value for now.

**Comment by derekwaynecarr** [comment]
It would be nice if you could relate the active deadline to this field:

https://github.com/kubernetes/kubernetes/blob/master/pkg/api/types.go#L1294

**Comment by spadgett** [comment]
I can humanize the time and show time elapsed

**Comment by spadgett** [comment]
@derekwaynecarr how does this look?

screen shot 2016-02-26 at 4 45 47 pm

**Comment by jwforres** [comment]
Trying to decide if i'd rather see elapsed or remaining

**Comment by spadgett** [comment]
@derekwaynecarr mentioned that the pod might not be killed exactly when the deadline is reached, which is why I went with elapsed

**Comment by jwforres** [comment]
Ah k, makes sense then

**Comment by derekwaynecarr** [comment]
Looks awesome!

File: assets/app/views/browse/_pod-details.html [hunk 1]
```diff
@@ -17,8 +17,15 @@
...
         <dt>Node:</dt>
         <dd>{{pod.spec.nodeName || 'unknown'}} <span ng-if="pod.status.hostIP && pod.spec.nodeName != pod.status.hostIP">({{pod.status.hostIP}})</span></dd>
-        <dt>Restart policy:</dt>
+        <dt>Restart Policy:</dt>
         <dd>{{pod.spec.restartPolicy || 'Always'}}</dd>
+        <dt ng-if-start="pod.spec.activeDeadlineSeconds">Active Deadline:</dt>
+        <dd ng-if-end>
+          {{pod.spec.activeDeadlineSeconds | humanizeDurationValue : 'seconds'}}
+          <span ng-if="pod.status.phase === 'Running'" class="text-muted">
+            (<duration-until-now timestamp="pod.status.startTime"></duration-until-now> elapsed)
```
Comment (jwforres) [inline assets/app/views/browse/_pod-details.html]: are we sure startTime will always be there? maybe the ng-if above should be `pod.status.phase === 'Running' && pod.status.startTime` ?

**Comment by jwforres** [comment]
just the one question, otherwise LGTM

**Comment by spadgett** [comment]
@jwforres updated

**Comment by jwforres** [comment]
[merge]

**Pull request merged**